drivers/display/display_ili9163.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022 Tavish Naruka <tavishnaruka@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_DRIVERS_DISPLAY_DISPLAY_ILI9163_H_
+#define ZEPHYR_DRIVERS_DISPLAY_DISPLAY_ILI9163_H_
+
+#include <device.h>
+
+/* Commands/registers. */
+#define ILI9163_GAMSET 0x26
+#define ILI9163_FRMCTR1 0xB1
+#define ILI9163_PGAMCTRL 0xE0
+#define ILI9163_NGAMCTRL 0xE1
+#define ILI9163_PWCTRL1 0xC0
+#define ILI9163_PWCTRL2 0xC1
+#define ILI9163_VMCTRL1 0xC5
+#define ILI9163_VMCTRL2 0xC7
+#define ILI9163_GAMADJ 0xF2
+#define ILI9163_DISPINV 0xB4
+
+/* Commands/registers length. */
+#define ILI9163_GAMSET_LEN 1U
+#define ILI9163_FRMCTR1_LEN 2U
+#define ILI9163_PGAMCTRL_LEN 15U
+#define ILI9163_NGAMCTRL_LEN 15U
+#define ILI9163_PWCTRL1_LEN 2U
+#define ILI9163_PWCTRL2_LEN 1U
+#define ILI9163_VMCTRL1_LEN 2U
+#define ILI9163_VMCTRL2_LEN 1U
+#define ILI9163_GAMADJ_LEN 1U
+#define ILI9163_DISPINV_LEN 1U
+
+/** X resolution (pixels). */
+#define ILI9163_X_RES 160U
+/** Y resolution (pixels). */
+#define ILI9163_Y_RES 128U
```

Comment:
Unnecessary, unused macros

Reply:
they are:

https://github.com/zephyrproject-rtos/zephyr/blob/aacf36c1fc0de6ea83e65218953222e0778baf12/drivers/display/display_ili9xxx.c#L438-L439

Reply:
okay, that is bad